.github/workflows/tests.yml

```diff
@@ -39,9 +39,6 @@ jobs:
         python -m pip install --upgrade pip
         pip install -e .
         pip install -r requirements/test.txt
-    - name: Test with pytest
+    - name: Run tests and check code format
       run: |
-        python3 -m pytest
-    - name: Lint with flake8
-      run: |
-        flake8 .
+        python3 -m pytest -ruff --ruff-format
```

Comment:
Is `-ruff` with a single dash really correct?

Reply:
Good catch! That's double for sure